historical.json

```diff
@@ -1258,6 +1258,78 @@
           "north america": 0.12,
           "pacific islands": 0.27
         }
+      },
+      "2015": {
+        "": {
```

Comment:
Why empty key?

Reply:
The empty keys are the column headers which don't have a title denoted to them. I don't know what to do with them yet, but they are something I'm looking into. Any ideas will also be appreciated. @kilemensi

Reply:
This is why we have Jean in the team @ascii-dev? Ask her what they mean and how are we to handle them.

Reply:
Did you discuss this with Jean @ascii-dev ?

Reply:
We got an answer for this @ascii-dev

Reply:
Oh... Yes... I had the discussion with Jean, it skipped me... I'll jump on fixing it immediately...